### PR TITLE
feat(callout with media): web component version

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#3747 

### Description

create callout with media web component

### Changelog

**New**

- component, tests, and stories

**Changed**

- updated styles

### CODE REVIEW

File: packages/web-components/src/components/callout-with-media/__stories__/callout-with-media.stories.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import '../callout-with-media';
+import '../callout-with-media-image';
+import '../callout-with-media-video';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import readme from './README.stories.mdx';
+
+const image = html`
+  <dds-callout-with-media-image
+    alt="Image alt text"
+    default-src="https://dummyimage.com/672x378/ee5396/161616&text=16:9"
```
Comment (asudoh) [inline packages/web-components/src/components/callout-with-media/__stories__/callout-with-media.stories.ts:22]:
```suggestion
    default-src="https://dummyimage.com/672x378/ee5396/161616&amp;text=16:9"
```

File: packages/web-components/src/components/callout-with-media/__tests__/callout-with-media.test.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    it('should render with image attributes', async function() {
+      render(
+        template({
+          copy: 'copy-foo',
+          heading: 'heading-foo',
+          children: html`
+            <dds-callout-with-media-image
+              alt="Image alt text"
+              default-src="https://dummyimage.com/672x378/ee5396/161616&text=16:9"
```
Comment (asudoh) [inline packages/web-components/src/components/callout-with-media/__tests__/callout-with-media.test.ts:42]:
```suggestion
              default-src="https://dummyimage.com/672x378/ee5396/161616&amp;text=16:9"
```

**Review by asudoh — COMMENTED** [review]
Good to see the progress @ariellalgilmore! A couple of suggestions to address the ESLint error. BTW any blockers to move this out of draft?

File: packages/web-components/src/components/callout-with-media/__stories__/callout-with-media.stories.ts (hunk 1 above)
Comment (ariellalgilmore) [reply packages/web-components/src/components/callout-with-media/__stories__/callout-with-media.stories.ts:22]: Thank you!!! And changing it to ready for review

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @ariellalgilmore!

**Review by jeffchew — CHANGES_REQUESTED** [review]
Hi @ariellalgilmore , can you add stable selectors and prop table to the documentation?

**Comment by ariellalgilmore** [comment]
@jeffchew updated! thank you!

**Comment by jeffchew** [comment]
@ariellalgilmore can you look to see what is happening here?

cc: @kennylam 

Screen Shot 2020-11-03 at 10 30 29 AM

**Comment by ariellalgilmore** [comment]
@jeffchew When comparing to the react version these changes seem to more closely align. Before some styles of pictogram weren't actually being applied. On the left is web components canary where for example margin-top is not actually being applied and on the right is from my PR where it is now applied causing these slight changes.
Screen Shot 2020-11-03 at 7 40 10 AM

**Review by kennylam — APPROVED** [review]
LGTM thanks @ariellalgilmore!

**Review by jeffchew — APPROVED** [review]

**Pull request merged**